### DISCUSSION TITLE
Proposal: Generating manifest.json and Package Linking

### DISCUSSION THREAD

**Discussion by Xonoroff** [AssetRipper/AssetRipper#910] [Ideas]
I would like to propose an idea to enhance the functionality of AssetRipper by introducing the generation of a manifest.json file and the ability to link packages to the exported project. 

The proposed functionality can be implemented within AssetRipper by extending its existing capabilities. The tool can be enhancedwith toggelt to automatically generate the manifest.json file during the asset extraction process. Additionally, a mechanism can be introduced to allow users to manually associate extracted assets with their corresponding Unity packages, either through user input or through automated package detection.

Here's screenshot from the windows game.
![image](https://github.com/AssetRipper/AssetRipper/assets/9823065/0b14c563-3f67-4156-9c23-28337cd7f7b6)

Here's result of the AssetRiper tool.
![image](https://github.com/AssetRipper/AssetRipper/assets/9823065/9d5b742d-be57-4f2e-87b7-b31090158c0b)

I would like to be able to avoid decompiling of such dlls from the tool and would like to have some mechanism to link packages from unity registry, or local packages directly to manifest without decompiling them.

**Comment by ds5678** [comment]
#805 will give you most of what you want.
